### PR TITLE
Added update script and updated README

### PR DESCRIPTION
**Pull request by msadang**
Users are experiencing errors when the notebook repo does not include all recent changes which may cause unnecessary issues to be made.  This script provides a method to update a local repo for a user on demand to ensure they are always working with the latest release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,6 @@ These notebooks provide examples of how to use RAPIDS.  These notebooks are desi
 
 This repository serves as a convenience for our developers and users as a colocation of all RAPIDS notebooks.
 
-To get the latest notebook updates, use the following command:
+To get the latest notebook repo updates, run the update.sh script or use the following command:
```
Comment (ajschmidt8) [inline README.md:18]:
```suggestion
To get the latest notebook repo updates, run `./update.sh` or use the following command:
```

File: update.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+git submodule update --init --remote --no-single-branch --depth 1
```
Comment (mike-wendt) [inline update.sh:1]: I would add `#!/bin/bash` at the start to be safe

Also add `git pull` before the submodule call so we have the latest content from the repo if this has been checked out previously

**Review by ajschmidt8 — APPROVED** [review]
looks good. one minor suggestion to use backticks around `./update.sh`.

File: README.md (hunk 1 above)
Comment (mike-wendt) [inline README.md:18]: I would add ticks around `update.sh` so it's easier to ready

**Review by mike-wendt — CHANGES_REQUESTED** [review]

File: update.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+git pull
+git submoidule update --init --remote --no-single-branch --depth 1
```
Comment (harrism) [inline update.sh:4]:
```suggestion
git submodule update --init --remote --no-single-branch --depth 1
```

**Review by harrism — CHANGES_REQUESTED** [review]
This doesn't really solve the problem we're having with this repo. THe problem is that the RAPIDS team is not updating the submodule links to the correct version at every release.  It would be great if we could automate that or make it part of the release process (ideally when notebooks `branch-0.18` is created, the links should automatically be updated to point at the corresponding branches of the submodules, if submodules are allowed to point at branches and not commits).

**Comment by msadang** [comment]
rerun test

**Comment by mike-wendt** [comment]
> This doesn't really solve the problem we're having with this repo. THe problem is that the RAPIDS team is not updating the submodule links to the correct version at every release. It would be great if we could automate that or make it part of the release process (ideally when notebooks `branch-0.18` is created, the links should automatically be updated to point at the corresponding branches of the submodules, if submodules are allowed to point at branches and not commits).

@harrism we already use [branches](https://github.com/rapidsai/notebooks/blob/branch-0.17/.gitmodules) for the submodules. The commits in this repo are the latest that were available when we made the update.

This is why we run 
```
git submodule update --init --remote --no-single-branch --depth 1
```
as that walks to the tip of the branches in all of the repos and provides the update without us having to constantly commit to this repo every night.

The `update.sh` script is just a way for people like DSWS and others to stay on `main` branch and then run `./update.sh` to get the next release and the latest notebooks.

**Review by mike-wendt — APPROVED** [review]

**Pull request merged**